staging/src/k8s.io/apiextensions-apiserver/pkg/client/informers/externalversions/factory.go

```diff
@@ -169,6 +169,7 @@ func (f *sharedInformerFactory) InformerFor(obj runtime.Object, newFunc internal
 // API group versions.
 type SharedInformerFactory interface {
 	internalinterfaces.SharedInformerFactory
+	ForStoppableResource(resource schema.GroupVersionResource) (*StoppableInformerInfo, bool)
```

Comment:
Is the second `bool` return value redundant? If `*StoppableInformerInfo` == nil it implies not exist.

Another question is, from the way of function naming, `ForStoppableResource` and `ForResource` are similar, they should be no dependencies with each other. What about give  `ForStoppableResource` a new name, such as `GetStoppableInformer`, or call `ForResource` automatically?

Reply:
Yea, I think originally there was some confusion around a situation where the informer existed but was not started. I have modified `ForStoppableResource` to call `ForResource` initially. This should make the naming choice more sensible and got rid of the bool.